tests/database_test_utils.py

```diff
@@ -0,0 +1,113 @@
+from typing import Any
+
+from sqlalchemy import text
+from sqlmodel import Session, SQLModel
+from tad.core.db import get_engine
+
+
+class DatabaseTestUtils:
+    """
+    Class to use for testing database calls. On creation, this class destroys and recreates the database tables.
+    """
+
+    def __init__(self):
+        self.clear()
```

Comment:
I think the init should do the create all and the __del__ function should do the drop_all

Reply:
Because I want each test to run on its own and not depend on other things that may or may not have happened, the design is on purpose. If a function needs a database, you get an empty one and need to fill it yourself. The drop is done because I don't really want to think about 'what to clean' or 'what auto increment may change'.